config/crd/bases/jaegertracing.io_jaegers.yaml

```diff
@@ -6551,6 +6551,11 @@ spec:
                 additionalProperties:
                   type: string
                 type: object
+              metrics:
```

Comment:
This functionality is only implemented in the jaeger-query component, no other component should be aware of querying metrics. 

Given that, do you think it would make more sense to move it under `query`? I think it may have implications for `allInOne` as well. 

I'm pretty new to this repo, so I'm not across what are the best practices for the Jaeger CRD design, just thought to bring it up as a consideration as it's not the same as span storage where the collector also writes to a backing store.

Having said that, I believe @pavolloffay mentioned there's a chance the otel collector's exporters could be brought into Jaeger as part of the v2 effort, meaning the possibility of writing to metrics storage, in which case it's okay as it is here.

Reply:
I'm pretty new to this repo, either. This file is generated by `make bundle`, So, I'm not sure this is the right changes.

Reply:
Thanks for your review, This file is generated by `make bundle` command, I moved `JaegerMetricsStorageSpec` under `query` and `allInOne` Spec. and I re-run `make bundle`. That's may works right.